index.html

```diff
@@ -7724,6 +7724,429 @@
+      <code><a>RTCIceTransport</a></code> and a sequence of
+      <code><a>RTCCertificate</a></code> objects. If <code>certificates</code> is
+      non-empty, check that the <code>expires</code> attribute of each
+      <code><a>RTCCertificate</a></code> object is in the future. If a certificate has
+      expired, throw an InvalidParameter exception; otherwise, store the
+      certificates.</p>
+      <p>A newly constructed <code><a>RTCQuicTransport</a></code> <em class="rfc2119"
+      title="MUST">MUST</em> listen and respond to incoming QUIC packets before
+      <code>start()</code> is called. However, to complete the negotiation it is
+      necessary to verify the remote fingerprint, which is dependent on
+      <code>remoteParameters</code>, passed to <code>start()</code>. To verify the remote
+      fingerprint, compute the fingerprint <var>value</var> for the selected remote
+      certificate using the signature digest algorithm, and compare it against
+      <code>remoteParameters.fingerprints</code>. If the selected remote certificate
+      <code>RTCQuicFingerprint.value</code> matches
+      <code>remoteParameters.fingerprints[j].value</code> and
+      <code>RTCQuicFingerprint.algorithm</code> matches
+      <code>remoteParameters.fingerprints[j].algorithm</code> for any value of
+      <var>j</var>, the remote fingerprint is verified. After the QUIC handshake exchange
+      completes (but before the remote fingerprint is verified) incoming media packets
+      may be received. A modest buffer <em class="rfc2119" title="MUST">MUST</em> be
+      provided to avoid loss of media prior to remote fingerprint validation (which can
+      begin after <code>start()</code> is called).</p>
+      <p>If an attempt is made to construct a <code><a>RTCQuicTransport</a></code>
+      instance from an <code><a>RTCIceTransport</a></code> in the <code>closed</code>
+      state, an <code>InvalidStateError</code> exception is thrown. Since the 
+      QUIC negotiation occurs between transport endpoints
+      determined via ICE, implementations of this specification <em class="rfc2119"
+      title="MUST">MUST</em> support multiplexing of STUN, TURN, DTLS, QUIC and RTP and/or
+      RTCP.</p>
+      <p>An <code><a>RTCQuicTransport</a></code> object in the <code>closed</code> or
+      <code>failed</code> states can be garbage-collected when it is no longer
+      referenced.</p>
+    </section>
+    <section id="rtcquictransport-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">typedef (octet[]) ArrayBuffer;</pre>
+        <div class="idlTypedefDesc"></div>
+      </div>
+      <div>
+        <pre class="idl">
+        [ Constructor (RTCIceTransport transport, sequence&lt;RTCCertificate&gt; certificates)]
+interface RTCQuicTransport : RTCStatsProvider  {
+    readonly        attribute FrozenArray&lt;RTCCertificate&gt; certificates;
```

Comment:
Shouldn't an RTCQuicTransport also be an RTCDataTransport so RTCDataChannels can be used with it?